### PR TITLE
Apply full GPU-relative WIC color-match patch

### PR DESCRIPTION
**Pull request by xmarre**
## Summary
- apply the provided wic_color_match_full_gpu_relative.patch
- include all patch changes in person_capture/gui_app.py

## Notes
- patch was applied after pulling latest master
- changes were handed off to a dedicated worktree branch before commit

## Summary by CodeRabbit

* **Performance Improvements**
  * Color-matching now runs at full resolution on the GPU for faster, more consistent results.
* **Bug Fixes / Accuracy**
  * Changed-pixel detection is now produced alongside the processed image for more accurate change reporting; CPU fallback still available.
* **Documentation**
  * UI/help text updated to reflect the expanded GPU processing and remaining CPU-bound steps.

### CODE REVIEW

File: person_capture/gui_app.py [hunk 1]
```diff
@@ -9122,55 +9136,88 @@ def _wic_yuv444_color_match_gpu_allowed(self, pixels: int, *, lowfreq: float) ->
...
+
+                # OpenCV's uint8 BGR<->YCrCb uses JPEG YCrCb.  This float CUDA
+                # implementation follows the same display-referred transform;
+                # small +/-1 LSB differences versus OpenCV's integer kernels are
+                # acceptable and avoid the previous full-resolution CPU cvtColor.
+                y = (0.114 * b) + (0.587 * g) + (0.299 * r)
+                cr = ((r - y) * 0.713) + 128.0
+                cb = ((b - y) * 0.564) + 128.0
+                ycc_u8 = torch.clamp(torch.round(torch.stack((y, cr, cb), dim=2)), 0, 255).to(torch.uint8)
+                ycc_f = ycc_u8.to(torch.float32)
```
[the remark this answers is not in the document]
Comment (xmarre) [reply person_capture/gui_app.py:9188]: Implemented in eff5acc. I validated the concern against this path: we explicitly allow +/-1 LSB CUDA/OpenCV conversion drift, so counting any nonzero delta could force false-positive rewrites on no-op remaps. The changed mask now counts only per-pixel max-channel deltas > 1 (`diff > 1`).

**Comment by xmarre** [comment]
Addressing the non-inline review note as well: I reviewed the outside-diff CodeRabbit summary/review status comment and applied the only actionable code finding in commit eff5acc (changed-pixel threshold now ignores +/-1 LSB CUDA rounding drift). I did not add unrelated docstring-churn here because this PR scope is the GPU color-match behavior; if you want, I can open a follow-up PR dedicated to docstring coverage.
